src/DockerRunner.jl

```diff
@@ -93,9 +93,18 @@ function DockerRunner(workspace_root::String;
     push!(workspaces, compiler_wrapper_path => "/opt/bin")
 
     # Generate CMake and Meson files
-    generate_toolchain_files!(platform, envs; toolchains_path=toolchains_path)
```

Comment:
Side note, the constructors of the two runners have plenty of duplication :disappointed:

Reply:
Addressed by # 191